mint/git-clone/mint-leaf.yml

```diff
@@ -52,7 +52,9 @@ tasks:
       printf "${LATEST_SHA_CACHE_BUSTER}" >> "$MINT_VALUES/latest-sha-cache-buster"
     env:
       GIT_SSH_KEY: ${{ params.ssh-key }}
-      GITHUB_ACCESS_TOKEN: ${{ params.github-access-token }}
+      GITHUB_ACCESS_TOKEN:
+        value: ${{ params.github-access-token }}
+        cache: false
```

Comment:
If you specify a SHA we don't want to cache bust this just because your access token changes